src/handlers/edit.ts

```diff
@@ -24,8 +24,12 @@ export namespace EditHandler {
       task.description = args.description;
     }
 
-    if (typeof args.by === "string") {
+    if (typeof args.by === "string")
+    try {
       task.by = new Date(args.by).toISOString();
```

Comment:
Should this be added to the `add` command as well?

Reply:
That's true! Thanks for noticing. I fixed it! Since it was a small change, I pushed it to this PR